src/wp-includes/fonts/class-wp-font-family.php

```diff
@@ -0,0 +1,658 @@
+<?php
+/**
+ * Font Family class.
+ *
+ * This file contains the Font Family class definition.
+ *
+ * @package    WordPress
+ * @subpackage Fonts
+ * @since      6.4.0
+ */
+
+/**
+ * Font Library class.
+ *
+ * @since 6.4.0
+ */
+class WP_Font_Family {
+
+	/**
+	 * Font family data.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @var array
+	 */
+	private $data = array();
+
+	/**
+	 * WP_Font_Family constructor.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param array $font_data Font family data.
```

Comment:
This doesn't match the parameter name and type at all.

Reply:
This class no longer exists in [the revised feature's codebase](https://github.com/WordPress/gutenberg/tree/trunk/lib/compat/wordpress-6.5/fonts). All code in the feature has been through an initial code review in Gutenberg for compliance with Core documentation standards.